Arriba/Arriba.Web/webpack.config.js

```diff
@@ -25,8 +25,12 @@ module.exports = {
         ]
     },
     devServer : {
+        host: "0.0.0.0",
+        port: 80,
```

Comment:
Are you able to bind to multiple ports? It might be nice to keep 8080 and 80, so there's an option for anyone who has port 80 bound and old documentation will still be right. =)